WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderViewModel.kt

```diff
@@ -44,6 +44,9 @@ class ReaderViewModel @Inject constructor(
     private val _selectTab = MutableLiveData<Event<TabPosition>>()
     val selectTab: LiveData<Event<TabPosition>> = _selectTab
 
+    private val _forceTabSelection = MutableLiveData<Event<ReaderTag>>()
```

Comment:
What if we reused the existing `_selectTab = MutableLiveData<Event<TabPosition>>()` ? The nice thing would be that we'd move the responsibility of mapping `tab -> position` from the view into the VM. Wdyt?

Reply:
> What if we reused the existing _selectTab = MutableLiveData<Event<TabPosition>>() ?
I had considered that, but decided against it until the majority of the refactor work was complete. I could just as easily look up the `ReaderViewModel` state in the `ReaderPostListViewModel` to grab the tag tab index.

Reply:
> P.S. I'd consider writing a simple unit test for the changes made in ReaderViewModel. Wdyt?

Agreed, just wanted to put out a PR with where I was before continuing. :)

Reply:
> The other possible approach I was considering would be to use Android navigation 

The main reason I didn't go in this direction was that I wanted the animation to remain the same as if the tab was tapped directly. Just feels smoother. I can go either direction if you want it to change.

Reply:
>The main reason I didn't go in this direction was that I wanted the animation to remain the same as if the tab was tapped directly. Just feels smoother. I can go either direction if you want it to change.

I'm not sure what you mean. You can play the animation the same way you are playing it now. The only difference would be from where you receive the information that you should change the tab. Can you please elaborate. Thanks!

>I had considered that, but decided against it until the majority of the refactor work was complete. 

I'm just curious what your thoughts on this were. I might be missing something but it seems to me we are creating a duplicate and the only difference is that one LiveData accept ReaderTab and the other position. Right?

>I could just as easily look up the ReaderViewModel state in the ReaderPostListViewModel to grab the tag tab index.

We could also find it within `fun emptyStateTabChange(tag: ReaderTag)` right? So the ReaderViewModel would do derive the position from its internal state, wdyt?

Reply:
We've discussed all these points during the call, so I'm resolving this discussion ;).